zebra-network/src/address_book.rs

```diff
@@ -126,17 +133,31 @@ impl AddressBook {
     }
 
     /// Get the local listener address.
-    pub fn get_local_listener(&self) -> MetaAddrChange {
+    ///
+    /// This address contains minimal state, but it is not sanitized.
+    pub fn get_local_listener(&self) -> MetaAddr {
         MetaAddr::new_local_listener(&self.local_listener)
+            .into_new_meta_addr()
```

Comment:
This was a bit confusing at first, because it took me a while to realize that `MetaAddr::new_local_listener` returns a `MetaAddrChange`, and not a `MetaAddr`. Not sure if this is worth trying to make it clearer in the future :thinking:

Reply:
The other change methods all end in `_change`, so this is an easy fix 8434ad37